### PR TITLE
Use `ConfigId` from `mc-sgx-core-types`

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
Previously the `ConfigId` was defined in `mc-attest-core`. Now the
`ConfigId` from `mc-sgx-core-types` is used.

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3434** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3434" target="_blank">Graphite</a> 
    * **PR #3435** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3435" target="_blank">Graphite</a> 
      * **PR #3438** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3438" target="_blank">Graphite</a> 
        * **PR #3439** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3439" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3439?utm_source=stack-comment).

File: attest/core/src/types/report_body.rs [hunk 1]
```diff
@@ -235,22 +235,16 @@ impl Ord for ReportBody {
...
-                                                            .cpu_security_version()
-                                                            .cmp(&other.cpu_security_version())
-                                                        {
-                                                            Ordering::Equal => self
-                                                                .report_data()
-                                                                .cmp(&other.report_data()),
-                                                            ordering => ordering,
-                                                        },
-                                                        ordering => ordering,
+                                            match self.0.config_svn.cmp(&other.0.config_svn) {
```
Comment (nick-mobilecoin) [inline attest/core/src/types/report_body.rs:238]: Added task to #3437 to capture this may need to be brought over if `Ord` needs to persist.

**Review by dolanbernard — APPROVED** [review]

**Review by eranrund — APPROVED** [review]

**Comment by nick-mobilecoin** [comment]
@nick-mobilecoin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3439).

**Comment by nick-mobilecoin** [comment]
[Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3439) rebased this pull request as part of a merge.

**Comment by nick-mobilecoin** [comment]
@nick-mobilecoin merged this pull request with [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/3439).

**Pull request merged**